### ISSUE TITLE
State of the development

### ISSUE THREAD

**Issue by papoteur-mga** [Scrabble3D/4.0#6]
Hello,
Could you add on the README what is the state of the development, what is supposed to work and what not?

**Comment by Scrabble3D** [comment]
Good idea. Added a lot of todo's in the code, many ideas or where improvements are needed. Basics should work fine. But the README is needed anyway.
